### ISSUE TITLE
Quick access to selecting audio/video devices

### ISSUE THREAD

**Issue by ara4n** [element-hq/element-call#255]
Originally we had a chevron menu on the mute buttons to let you select audio and video capture devices, much as Jitsi does.

i *really* miss this - having to spelunk in the settings modal is super clunky for such a common operation as checking your current mic and selecting the right one.

please can we get the inline menu back?

**Comment by SimonBrandner** [comment]
It got removed to better support touch devices and to avoid having two different ways to do this

**Comment by ara4n** [comment]
yup, i know, and i think it was an error, especially as desktop devices are probably still the primary platform :)

**Comment by daniellekirkwood** [comment]
we have chevrons on the buttons now! we have further iterations to make but as this ticket is old and we've achieved the basics, I'll close it out.

It can be re-opened if there's something i missed...
